build.sbt

```diff
@@ -40,7 +40,7 @@ lazy val standardSettings = Defaults.defaultSettings ++ Seq(
     "-language:higherKinds",
     "-language:implicitConversions",
     "-unchecked",
-    "-Xfatal-warnings",
+    //"-Xfatal-warnings", temporarly removed in order for the whole thing to build. If we could change this setting to be selectively applied in the build script, then it might make sense to remove it permanently here, anyway...
```

Comment:
For metadata and data, they are all done, not for the other sub-services. I have a second copy of the repo on master opened on my second screen for translation purposes which I also keep up to date as a way to pick up all the latest changes as soon as they happen. Will still have to to second pass to make sure nothing got added to files after they were translated.